server/mediaserver.go

```diff
@@ -105,22 +104,28 @@ type authWebhookResponse struct {
 
 func NewLivepeerServer(rtmpAddr string, httpAddr string, lpNode *core.LivepeerNode) *LivepeerServer {
```

Comment:
I think there is no need to pass `httpAddr` to `NewLivepeerServer`, because it is not used in it.

Reply:
Do you think we should change `lpmscore.LPMSOpts` to not include `httpAddr`?

Reply:
Yes. If you look at `LPMS` struct, it uses `HttpAddr` only in case if `HttpMux` is null. But now we always setting `HttpMux` to new `Mux`, so `HttpAddr` is not used by `LPMS`.

Reply:
I see that now. Thanks @darkdarkdragon !

Reply:
Seems like there was a misunderstanding. 

> Do you think we should change lpmscore.LPMSOpts to not include httpAddr?

I understood this question to mean that the `HttpAddr: httpAddr` should be removed from the specific instantiation of LPMSOpts within `mediaserver.go`, as in

https://github.com/livepeer/go-livepeer/blob/81414c252679a8af38b0c81757071bbbb9533ac9/server/mediaserver.go#L106-L112

Not that feature of configuring the HTTP binding address should be removed from LPMS completely, eg https://github.com/livepeer/go-livepeer/pull/1036#discussion_r315778381